### ISSUE TITLE
Infectiousness profile eventualy needs an update for B117 mutation

### ISSUE THREAD

**Issue by Andiadm** [corona-warn-app/cwa-documentation#531]
The Infectiousness profile was based on He et al and later updated.
Recently a [Harvard publication](https://dash.harvard.edu/handle/1/37366884) suggest that the mean duration of the proliferation phase was 5.3 days in the B117 variant group vs. 2.0 days for non-B.1.1.7 virus.
I wanted to raise this issue because with the B117 becoming more and more widely spread, this finding obviously calls for a change in the profile.
Ideally, the CWA would use a differnt profile depending if the positively tested person was infected by B117. This might be a too big change. But I would prefer to produce more warning messages than too little.

**Comment by MikeMcC399** [comment]
There is a wishlist discussion about this topic.
See https://github.com/corona-warn-app/cwa-wishlist/issues/328

Also the blog entry https://www.coronawarn.app/en/blog/2021-02-23-corona-warn-app-risk-calculation-optimization/ which deals with reducing the filter from 10 minutes down to 5 minutes, for considering encounters, contains the following final relevant paragraph:

"The project participants are continuously working on improving the measurements and the resulting configuration parameters. For example, the experts also discuss new findings on the spread and infectivity of newly emerging virus mutations (so-called "variants of concern"). As soon as the project members further develop and adapt the risk calculation with regard to distances and signal attenuation, you will find more information here."

**Comment by Andiadm** [comment]
Thanks for pointing at that.
My issue here however is different, as I am suggesting a change to the  Infectiousness profile which is basically the factor applied to an exposure depending on the day before symtom onset.
Its not about the duration filter, distances or attenuation.

**Comment by MikeMcC399** [comment]
@Andiadm 
> My issue here however is different, as I am suggesting a change to the Infectiousness profile which is basically the factor applied to an exposure depending on the day before symtom onset the exposure happened.
Its not about the duration filter, distances or attenuation.

I think you are talking about the TRL (Transmission Risk Level). Is that correct?

The latest material about that is on https://www.coronawarn.app/assets/documents/cwa-rc3.pdf and explained in the video on https://www.coronawarn.app/en/blog/2020-12-30-cwa-behind-the-scenes/.

I suggest you either add your points onto the existing wishlist item or get this one transferred to the wishlist.

**Comment by Andiadm** [comment]
Yes, in essence the table in the middle of page 15 in the PDF which I am quite familiar with.
Which whishlist do you mean?

**Comment by MikeMcC399** [comment]
@Andiadm 
> Yes, in essence the table in the middle of page 15 in the PDF which I am quite familiar with.
Which whishlist do you mean?

Good to hear that we are both looking at the same page!

There is a different repository which collects wishlist items. It is located at https://github.com/corona-warn-app/cwa-wishlist

The specific wishlist item was the one I previously referred to at https://github.com/corona-warn-app/cwa-wishlist/issues/328.

One of the moderators like @dsarkar can transfer this whole item to the wishlist, if that is what you would like. Just reply here and they will pick it up.

**Comment by Andiadm** [comment]
It would be better to transfer the entire thread to the oder repository than just adding it to the existing topic as the issue here is very specific.
Either ways are fine with me, @moderator, please pick up!
